src/pages/outreach.tsx

```diff
@@ -47,32 +76,70 @@ const About: NextPage = () => {
                 </Fade>
                 <Fade bottom duration={1000} delay={300} distance="30px">
                     <p>
-                        We have put a main focus on encouraging elementary and junior high students
-                        into the Computer Science field
-                        <br />
-                        and have been using the website Hour of Code to help kids gain the skills to
-                        already start programming on their own!
-                        <br />
-                        We have already visited multiple schools, such as Windsor Elementary School
-                        and Carman Collegiate. <br />
-                        Other schools are welcome to contact us by email at uofmwics@gmail.com!
+                        One of the main goals of WICS is to promote the field of computer science
+                        within our community.
                     </p>
                 </Fade>
-                <Fade bottom duration={1000} delay={400} distance="30px">
-                    <Button href="mailto:uofmwics@gmail.com" variant="contained" color="secondary">
-                        Contact us
-                    </Button>
-                </Fade>
             </div>
-
-            <Container component="main">
+            <Container component="main" maxWidth="sm">
```

Comment:
i actually changed to md to keep the margin consistency with other pages(they are usually md)